snake_n_ladders/lib/ladder.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+require_relative 'hop'
+
+class Ladder < Hop
+end
```

Comment:
shouldn't these be structs?
https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/choosing-between-class-and-struct?redirectedfrom=MSDN